CODEOWNERS

```diff
@@ -12,8 +12,9 @@
 /torch/autograd/ @albanD
 
 # Tensorpipe RPC Agent.
-/torch/csrc/distributed/rpc/tensorpipe_agent.cpp @jiayisuse @osalpekar @lw @beauby
-/torch/csrc/distributed/rpc/tensorpipe_agent.h @jiayisuse @osalpekar @lw @beauby
```

Comment:
@jiayisuse @osalpekar As I was touching this file I updated this, let me know if you'd rather remain enrolled.